ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
runtests already puts the output somewhere you can look at it, in a file. every modern text editor I've ever used (except maybe win32 notepad?) can handle unix newlines. Newline translation would break most of my tests that generate files, so I'm not adding it to write. You're welcome to add write_text or something that does platform-specific translation.

Next time I'm on windows I'll see whether the windows console needs crlf, but I'm not sure it does.